### ISSUE TITLE
Add alt attribute to images in html export

### ISSUE THREAD

**Issue by justwalked** [kensanata/mastodon-archive#114]
Images attached to toots have a description property, I think this information should go in the [alt attribute](https://developer.mozilla.org/fr/docs/Web/HTML/Element/img#alt) of the image. It's very important for accessibility, or if the images go missing.

**Comment by kensanata** [comment]
I agree!
